src/osha/oira/client/browser/settings.py

```diff
@@ -41,18 +39,16 @@ def my_sectors(self):
         # TODO
         return set()
 
-    @property
-    @memoize
-    def my_tools(self):
-        return {s.tool for s in self.my_sessions}
+    # @property
+    # @memoize
+    # def my_tools(self):
+    #     return {s.zodb_path for s in self.my_sessions}
```

Comment:
Why this was modified and commented?

Reply:
I want to prevent further reviews at this point. This is currently not called but may be handy later.